docs/Changelog.md

```diff
@@ -2,6 +2,7 @@
 
 1.4.0 (Not released yet)
 ------------------------
+* [#2992](https://github.com/TouK/nussknacker/pull/2992) Moved DeploymentComment validation to backend, added custom classes for different comments. 
```

Comment:
Chagelog is for users or administrators, not developers. There is no benefits from user perspective?

Reply:
there are, added them, thanks